Source/SumIQ.cpp

```diff
@@ -138,3 +138,137 @@ PeleC::sum_integrated_quantities()
 #endif
   }
 }
+
+void
+PeleC::monitor_extrema()
+{
+  BL_PROFILE("PeleC::sum_integrated_quantities()");
+
+  if (verbose <= 0) {
+    return;
+  }
+
+  bool local_flag = true;
+
+  int finest_level = parent->finestLevel();
+  amrex::Real time = state[State_Type].curTime();
+  amrex::Vector<std::string> extrema_vars = {
+    "density", "x_velocity", "y_velocity", "z_velocity",
+    "eint_e",  "Temp",       "pressure",   "massfrac"};
+
+  int nspec_extrema = 1;
+  if (!fuel_name.empty()) {
+    nspec_extrema += 1;
+    extrema_vars.push_back(fuel_name);
+  }
+  if (!flame_trac_name.empty()) {
+    nspec_extrema += 1;
+    extrema_vars.push_back(flame_trac_name);
+  }
+  if (!extrema_spec_name.empty()) {
+    nspec_extrema += 1;
+    extrema_vars.push_back(extrema_spec_name);
+  }
+  auto nextrema = extrema_vars.size();
+  constexpr amrex::Real neg_huge = std::numeric_limits<amrex::Real>::lowest();
+  constexpr amrex::Real huge = std::numeric_limits<amrex::Real>::max();
+  amrex::Vector<amrex::Real> minima, maxima;
+  minima.resize(nextrema);
+  maxima.resize(nextrema);
+  for (int ii = 0; ii < nextrema; ++ii) {
+    maxima[ii] = neg_huge;
+    minima[ii] = huge;
+  }
+
+  for (int lev = 0; lev <= finest_level; lev++) {
+    PeleC& pc_lev = getLevel(lev);
+    for (int ii = 0; ii < nextrema - nspec_extrema; ++ii) {
```

Comment:
why `ii` and not just `i`?

Reply:
I dislike single character variable names and try to avoid them to make search, find+replace, etc, easier to use.

Reply:
lol

Reply:
Is your editor the stickies app?